uint8 action
string limb
geometry_msgs/Pose pose
---
bool success